app/src/main/java/sergio/splashbaseviewer/adapters/ImageContainerAdapter.java

```diff
@@ -0,0 +1,77 @@
+package sergio.splashbaseviewer.adapters;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import sergio.splashbaseviewer.R;
+import sergio.splashbaseviewer.utils.models.SplashImage;
+import sergio.splashbaseviewer.utils.models.SplashImageResponse;
+
+/**
+ * @author s.ruiz
+ */
+
+public class ImageContainerAdapter extends BaseAdapter {
+
+    private List<SplashImage> images = new ArrayList<>();
+
+    public void setImages(List<SplashImageResponse> images) {
```

Comment:
You are using an retrofit response in the view. This is wrong.
You should receive this  -->  public void setImages(List<SplashImage> images)